docs/source/api.rst

```diff
@@ -61,6 +65,7 @@ Feature and Label Encoding (Single-GPU)
 
  .. autoclass:: cuml.preprocessing.LabelEncoder.LabelEncoder
     :members:
+    :inherited-members:
```

Comment:
Should we always be adding inherited-members?

Reply:
Good catch, I wasnt sure if this would change anything and was added as a quick test that I forgot to remove.

On another note: I think there is a bigger discussion to be had about inherited members in the documentation. Its currently disabled globally in `conf.py` with `numpydoc_show_inherited_class_members = False`. However, this information could be useful to have.

Reply:
Resolved